doc/source/examples/03_modeling/scale_map_bodies.mystnb

```diff
@@ -0,0 +1,146 @@
+---
+jupytext:
+  text_representation:
+    extension: .mystnb
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.16.1
+kernelspec:
+  display_name: .venv
+  language: python
+  name: python3
+---
+
+# Design: Scale and Map Bodies
+
+The purpose of this notebook is to demonstrate the `map` and `scale` functions and their usage for transforming bodies.
+
+```{code-cell} ipython3
+# Imports
+import numpy as np
+from ansys.geometry.core import Modeler
+from ansys.geometry.core.math.point import Point2D, Point3D
+from ansys.geometry.core.sketch.sketch import Sketch
+from ansys.geometry.core.math.frame import Frame
+from ansys.geometry.core.math.vector import UnitVector3D
+```
+
+```{code-cell} ipython3
+# Initialize the modeler for this example notebook
+m = Modeler()
+```
+
+## Scale Body
```

Comment:
Only thoughts on the scale body section: maybe it would be better to make copies and scale the copies and plot them all together so the user can see the difference (plotting one body at a time doesn't have an immediate visual difference between them)